keras_cv/bounding_box/converters.py

```diff
@@ -291,3 +291,36 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def clip_to_image(bounding_boxes, images, bounding_box_format):
```

Comment:
cool, once this is removed and the other PR is submitted this will LGTM

Reply:
Thanks @LukeWood waiting on https://github.com/keras-team/keras-cv/pull/607 and https://github.com/keras-team/keras-cv/pull/608!